src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -117,6 +98,45 @@ public void testGetUserStatusTransition() throws Exception {
         assertThat(returnValue.getToStatus(), is(equalTo(DE_ACTIVATED)));
     }
 
+    @Test
+    public void testUpdateUserVerificationStatus() throws Exception {
```

Comment:
Please add IT cases for exceptions as mentioned in the page https://confluence.site1.hyperwallet.local/display/PF/Verification+enforcement+through+REST

Reply:
added test cases . Thanks @Rrathinasabapath